source/kraken/tests/realtime_test.cpp

```diff
@@ -2740,3 +2740,317 @@ BOOST_AUTO_TEST_CASE(should_get_correct_base_stop_time_with_lollipop_II) {
+    BOOST_CHECK_EQUAL(res.response_type(), pbnavitia::NO_SOLUTION);
+    BOOST_CHECK_EQUAL(res.journeys_size(), 0);
+    BOOST_REQUIRE_EQUAL(pt_data.meta_vjs.size(), 1);
+    BOOST_REQUIRE_EQUAL(pt_data.meta_vjs.exists("vj_new_trip"), false);
+
+    // call function with is_realtime_add_trip_enabled=false
+    // the new trip update is blocked directly
+    navitia::handle_realtime("feed-1", timestamp, new_trip, *b.data, true, false);
+    b.make();
+    res = compute("20190101T073000", "stop_point:A", "stop_point:G");
+    BOOST_CHECK_EQUAL(res.response_type(), pbnavitia::NO_SOLUTION);
+    BOOST_CHECK_EQUAL(res.journeys_size(), 0);
+    BOOST_REQUIRE_EQUAL(pt_data.meta_vjs.size(), 1);
+    BOOST_REQUIRE_EQUAL(pt_data.meta_vjs.exists("vj_new_trip"), false);
+
+
+
+
+    // New call with all activation parameter = true
+    navitia::handle_realtime("feed-1", timestamp, new_trip, *b.data, true, true);
+    b.make();
+
+    // Check meta vj table
+    BOOST_REQUIRE_EQUAL(pt_data.meta_vjs.size(), 2);
+    BOOST_REQUIRE_EQUAL(pt_data.meta_vjs.exists("vj:1"), true);
+    auto* mvj = pt_data.meta_vjs.get_mut("vj:1");
+    BOOST_CHECK_EQUAL(mvj->get_label(), "vj:1");
+    BOOST_CHECK_EQUAL(mvj->get_base_vj().size(), 1);
+    BOOST_CHECK_EQUAL(mvj->get_adapted_vj().size(), 0);
+    BOOST_CHECK_EQUAL(mvj->get_rt_vj().size(), 0);
+    BOOST_REQUIRE_EQUAL(pt_data.meta_vjs.exists("vj_new_trip"), true);
+    mvj = pt_data.meta_vjs.get_mut("vj_new_trip");
+    BOOST_CHECK_EQUAL(mvj->get_label(), "vj_new_trip");
+    BOOST_CHECK_EQUAL(mvj->get_base_vj().size(), 0);
+    BOOST_CHECK_EQUAL(mvj->get_adapted_vj().size(), 0);
+    BOOST_CHECK_EQUAL(mvj->get_rt_vj().size(), 1);
+
+    // Check vj table
+    BOOST_REQUIRE_EQUAL(pt_data.vehicle_journeys_map.size(), 2);
+    BOOST_REQUIRE_EQUAL(pt_data.vehicle_journeys.size(), 2);
+
+    auto* vj = pt_data.vehicle_journeys_map["vj:1"];
+    BOOST_CHECK_EQUAL(vj->uri, "vj:1");
+    BOOST_CHECK_EQUAL(vj->idx, 0);
+    BOOST_CHECK_EQUAL(vj->meta_vj->get_label(), "vj:1");
+    BOOST_CHECK_EQUAL(vj->stop_time_list.size(), 4);
+    BOOST_CHECK_EQUAL(vj->base_validity_pattern()->days, year("11111111"));
+    BOOST_CHECK_EQUAL(vj->adapted_validity_pattern()->days, year("11111111"));
+    BOOST_CHECK_EQUAL(vj->rt_validity_pattern()->days, year("11111111"));
+
+    vj = pt_data.vehicle_journeys_map["vj_new_trip:modified:0:feed-1"];
+    BOOST_CHECK_EQUAL(vj->company->uri, comp_uri);
+    BOOST_CHECK_EQUAL(vj->company->name, comp_name);
+    BOOST_CHECK_EQUAL(vj->uri, "vj_new_trip:modified:0:feed-1");
+    BOOST_CHECK_EQUAL(vj->idx, 1);
+    BOOST_CHECK_EQUAL(vj->meta_vj->get_label(), "vj_new_trip");
+    BOOST_CHECK_EQUAL(vj->stop_time_list.size(), 4);
+    BOOST_CHECK_EQUAL(vj->base_validity_pattern()->days, year("00000000"));
+    BOOST_CHECK_EQUAL(vj->adapted_validity_pattern()->days, year("00000000"));
+    BOOST_CHECK_EQUAL(vj->rt_validity_pattern()->days, year("00000001"));
+
```

Comment:
If i understand, for a new added trip we have:
- base vj with pattern 0s
- adapted vj with pattern 0s and 
- rt vj with pattern 0s and 1 once.

Reply:
when we have an Add Trip we create one Meta VJ with one new VJ.
Indeed, this VJ has no **base** and **adapted** validity pattern but only a **RT** validity pattern.
In this example, we have only one disrupted day, that's the reason for _0000000**1**_